packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -85,69 +196,20 @@ Scenarios.fromProject(baseApp)
     project.linkDependency('babel-plugin-ember-test-metadata', {
       baseDir: __dirname,
     });
-
-    merge(project.files, {
-      tests: {
-        unit: {
-          'with-hooks-test.js': `import { module, test } from 'qunit';
-import { getTestMetadata } from '@ember/test-helpers';
-
-module('Acceptance | with-hooks-test', function (hooks) {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/with-hooks-test.js'));
-  });
-});
-`,
-          'without-hooks-test.js': `import { module, test } from 'qunit';
-import { getTestMetadata } from '@ember/test-helpers';
-
-module('Acceptance | without-hooks-test', function () {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/without-hooks-test.js'));
-  });
-});
-`,
-          'with-multiple-modules-test.js': `import {module, test} from 'qunit';
-import { getTestMetadata } from '@ember/test-helpers';
-
-module('Acceptance | with-multiple-modules-test', function (hooks) {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/with-multiple-modules-test.js'));
-  });
-});
-
-module('Acceptance | with-multiple-modules-test 2', function (hooks) {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/with-multiple-modules-test.js'));
-  });
-});
-`,
-        },
-      },
-    });
   })
   .forEachScenario((scenario) => {
     describe(scenario.name, () => {
+      const EMBROIDER = 'embroider';
+      const EMBROIDER_PATH_SEGMENT = '999999';
```

Comment:
maybe we could generate the SHA here instead of making it constant. that'd be a higher fidelity test case I think.